io_context.go

```diff
@@ -25,15 +30,36 @@ func (ic *IOContext) Closep() error {
 	return newError(C.avio_closep(&ic.c))
 }
 
+func (ic *IOContext) Close() error {
+	return newError(C.avio_close(ic.c))
+}
+
 func (ic *IOContext) Open(filename string, flags IOContextFlags) error {
 	cfi := C.CString(filename)
 	defer C.free(unsafe.Pointer(cfi))
 	return newError(C.avio_open(&ic.c, cfi, C.int(flags)))
 }
 
-func (ic *IOContext) Write(b []byte) {
+func (ic *IOContext) Write(b []byte) (int, error) {
 	if b == nil {
-		return
+		return 0, nil
 	}
 	C.avio_write(ic.c, (*C.uchar)(unsafe.Pointer(&b[0])), C.int(len(b)))
+	return len(b), nil
```

Comment:
This is misleading since we don't really know if the amount of written bytes is `len(b)` 🤔

Reply:
Yea, i realized this too while coding it, but didn't think it was a big deal. I think there might be a way to tell but i dont quite understand the C code here.

https://ffmpeg.org/doxygen/5.0/aviobuf_8c_source.html#l00232